packages/formation/sass/base/_va.scss

```diff
@@ -1247,34 +742,6 @@ html.touchevents {
   display: flex;
 }
 
-// Vertically aligns at top when flex-direction: row;
```

Comment:
These va-flex classes also appear in the dhp-connected-devices/components/PageNotFound.jsx file, are these set somewhere else?

Reply:
@powellkerry Good catch! Yes, I'm moving a group of app specific styles to the app here (same with the one above): https://github.com/department-of-veterans-affairs/vets-website/pull/27086